andela/ah-infinity-stones-frontend
# #161966890 Implement article bookmarks feature

#### What does this PR do?
- Implement article bookmarks

#### Description of Task to be completed?
- Implement bookmarking feature
- Implement un-bookmarking feature
- Write tests

#### How should this be manually tested?
1. `$ git clone -b ft-bookmark-#161966890 https://github.com/andela/ah-infinity-stones-frontend.git`  and `$ cd ah-infinity-stones-frontend`
2. Install the dependencies - `$ npm install`
3. Start the app by running `$ npm start` - This will run the app in the development mode.  
 To test the feature. Open http://localhost:3000/login  to view it in the browser. Provide email and password and click submit. This assumes that you have visited `http://localhost:3000/signup`, created and activated your account.
4. Visit http://localhost:3000  and select an article button. At the bottom of the article is a bookmark icon that will bookmark an article when clicked.
4. Test the app by running `$ npm test`
5. Check test coverage by running `$ npm test -- --coverage`

#### What are the relevant pivotal tracker stories?
[#161966890](https://www.pivotaltracker.com/story/show/161966890)

#### Screenshots
<img width="1050" alt="screenshot 2019-01-30 at 19 49 16" src="https://user-images.githubusercontent.com/17976289/51997636-2edff880-24c8-11e9-8a5a-634dcd0dbdf7.png">
<img width="1050" alt="screenshot 2019-01-30 at 19 46 51" src="https://user-images.githubusercontent.com/17976289/51997480-df99c800-24c7-11e9-9d9c-b2b6fab17a44.png">
<img width="1050" alt="screenshot 2019-01-30 at 19 48 10" src="https://user-images.githubusercontent.com/17976289/51997562-0952ef00-24c8-11e9-91a5-d7a66efbd7c8.png">
